setup-chainctl/README.md

```diff
@@ -32,7 +32,7 @@ permissions:
 steps:
 - uses: chainguard-dev/actions/setup-chainctl@main
   with:
-    environment: big-bird
-    audience: elmo
+    environment: example.com
+    audience: some-other-audience
```

Comment:
I'd make the sample just not specify these, since it is really only relevant to Chainguard folks (now)